feat(curriculum): add block 2 dialogs and tasks of the B1 course of the English Curriculum (#57471)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/24-b1-english-for-developers/learn-how-to-talk-about-past-experiences/671f7708379ef00668b843d4.md

---
id: 671f7708379ef00668b843d4
title: Task 98
challengeType: 19
dashedName: task-98
---

<!-- (Audio) Maria: I remember you would often stay late, fine-tuning the code until it was just right. -->

# --instructions--

Listen to the audio and answer the question below.

# --questions--

## --text--

> Did Brian use to work regular hours to adjust the code?

Reviewer: ```suggestion
Did Brian work regular hours to adjust the code?
```

Less clunky.

Author: This one is on purpose since the idea is to introduce 'used to' and its negative and interrogative forms.

## --answers--

No, he would often stay late fine-tuning the code.

---

Yes, he only worked regular hours to adjust the code.

### --feedback--

Maria explicitly mentions that Brian would often stay late, which suggests he worked beyond regular hours.

---

No, he only adjusted the code during meetings.

### --feedback--

Maria describes Brian staying late to fine-tune the code, not working on it exclusively during meetings.

---

Yes, but only when there was an urgent task.

### --feedback--

Maria indicates that staying late was a regular habit for Brian, not something limited to urgent tasks.

## --video-solution--

1

# --explanation--

> The word `would` in this context expresses a habitual action or repeated behavior in the past. It is often used to describe actions that someone regularly performed, similar to `used to`. 
> 
> Example: `She would always read a book before going to bed.` (reading a book before goig to bed was a common action for her)  
> 
> `Would`, however, focuses more on the action itself, while `used to` can describe states or habits. In the sentence `She used to live in New York before moving here.`, you would not use `would` instead of `used to`, since living in a place does not express an action but a state of things, not something you would habitually do.
> 
> In this sentence, Maria uses `would` to describe Brian’s repeated habit of staying late to fine-tune the code, which emphasizes his dedication.

Reviewer 2: ```suggestion
`Would` in this context expresses a habitual action or repeated behavior in the past. It is often used to describe actions that someone regularly performed, similar to `used to`. 

However, `would` focuses more on the action itself, while `used to` can describe states or habits. For example:

- `She would always read a book before going to bed.` - Reading a book before going to bed was a common action for her.  

`She used to live in New York before moving here.` - here `used to` is used because living in a place expresses a state of things not an action or something you would habitually do.

In this sentence, Maria uses `would` to describe Brian’s repeated habit of staying late to fine-tune the code, which emphasizes his dedication.
```

I kept the explanation about used to, but I don't think this task should cover used to because the other dialogues already covered it in details.
Even with the changes, I think this explanation can be confusing and should be revisited.
